packages/migrate/test/fixtures/migrate/2790-import-2.ts.output

```diff
@@ -0,0 +1,11 @@
+export class Employee {
+    name: string;
+    badgeNumber?: string;
+    constructor({ name, badgeNumber }: {
```

Comment:
Output has broken formatting - all empty lines been removed. 
This can be issue of using ts printer... or the way plugins run currently.

Reply:
Good catch. Removed the printer and the formatting is fixed.